### PR TITLE
sig-cluster-lifecycle: add annual report for 2021

### PR DESCRIPTION
**Pull request by neolit123**
add annual report for sig-cluster-lifecycle for 2021
https://github.com/kubernetes/community/blob/master/committee-steering/governance/annual-reports.md

### CODE REVIEW

**Comment by neolit123** [comment]
@kubernetes/sig-cluster-lifecycle 
/assign @dims
/cc @timothysc @justinsb @fabriziopandini

File: sig-cluster-lifecycle/annual-report-2021.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  * Some projects like etcdadm, cluster-addons and kubeadm are currently looking for more contributors.
+  * We use the common methods of
+  [“help-wanted” labels](https://github.com/kubernetes-sigs/cluster-api/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+  and announcing the request for help on slack / mailing list.
+  * For the main SIG (e.g. when reviewing KEPs) the lead who has the time usually reviews, but we try to notify everyone
+  so that they can comment if they have the time too.
+* Is there a healthy onboarding and growth path for contributors in your SIG?
+What are some activities that the group does to encourage this? What programs are you participating in to grow contributors
+throughout the contributor ladder?
+  * One of our methods is to just record onboarding videos / hold Zoom sessions discussing a certain problem / area:
```
Comment (fabriziopandini) [inline sig-cluster-lifecycle/annual-report-2021.md:59]: What about adding that most of the meetings have a dedicated session to welcome newcomers?

File: sig-cluster-lifecycle/annual-report-2021.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    * [Cluster API stats](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=d7&var-metric=contributions&var-repogroup_name=SIG%20Cluster%20Lifecycle%20(Cluster%20API)&var-companies=All)
+    * [Overall Cluster Lifecycle stats](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=d7&var-metric=contributions&var-repogroup_name=SIG%20Cluster%20Lifecycle&var-companies=All)
+  * Everyone can contribute as long as they have the motivation and their ideas are good.
+
+## Current initiatives and project health
+
+* [x] Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members.
+* What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway?
+What are some of the longer tail projects that your group is working on?
+  * We did an KEP to standardize how clusters define insecure local container registries:
```
Comment (fabriziopandini) [inline sig-cluster-lifecycle/annual-report-2021.md:78]:
```suggestion
  * We did a KEP to standardize how clusters define insecure local container registries:
```
nit

File: sig-cluster-lifecycle/annual-report-2021.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    * [1755-communicating-a-local-registry](https://git.k8s.io/enhancements/keps/sig-cluster-lifecycle/generic/1755-communicating-a-local-registry)
+  * Cluster API has its own spin of the KEP process and has a number of interesting active proposals:
+    * [Cluster API proposals](https://github.com/kubernetes-sigs/cluster-api/tree/master/docs/proposals)
+  * Most of our subprojects have existed for a long time - including, kubeadm, kops, minikube, etc.
+* Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?
+  * Only kubeadm features go through the Alpha->GA stages of the Kubernetes release process:
+    * [keps/sig-cluster-lifecycle](https://https://git.k8s.io/enhancements/keps/sig-cluster-lifecycle)
+  * Some of the newer subprojects like Cluster API / etcdadm do have KEPs which we try to update.
```
Comment (fabriziopandini) [inline sig-cluster-lifecycle/annual-report-2021.md:86]:
```suggestion
  * Some of the newer subprojects like Cluster API / etcdadm do have umbrella KEPs in the enhancement repo which we try to update.
```

File: sig-cluster-lifecycle/annual-report-2021.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  * Projects like kops, minikube, kubeadm, kubespray are mostly stable.
+  * Projects like Cluster API and etcdadm are alpha.
```
Comment (fabriziopandini) [inline sig-cluster-lifecycle/annual-report-2021.md:88]:
```suggestion
  * Projects like Cluster API and etcdadm are alpha and moving towards graduation following a process similare to Kubernetes.
```

**Review by fabriziopandini — COMMENTED** [review]
@neolit123 thanks for driving this effort!
only few nits from my side but overall the content is great

**Review by timothysc — APPROVED** [review]
/lgtm
/approve
/hold 

for comments^

**Comment by neolit123** [comment]
updated after @fabriziopandini 's review.

**Comment by dims** [comment]
/hold cancel

this looks ready

**Pull request merged**